glottolog3/__init__.py

```diff
@@ -140,6 +140,10 @@ def main(global_config, **settings):
         '/bp/search',
         views.languages,
         renderer='language/bpsearch_html.mako')
+    config.add_route_and_view(
+        'glottolog.bpapisearch',
+        'bp/api/search',
```

Comment:
Can we make this endpoint more like `GET /languoids?search={search string}` ? See Leon's PR where we specify the request type: https://github.com/uwblueprint/glottolog3/pull/8

And then your view function has to find the search param.

Reply:
done